.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,30 @@
+name: Bug report
+description: Things didn't meet expectations
+labels: 'C-bug'
+body:
+  - type: textarea
+    attributes:
+      label: Steps to reproduce the bug with the above code
+      description: A command like `cargo run -- options...` or multiple commands.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Actual Behaviour
+      description: When I do like *this*, *that* is happening and I think it shouldn't.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behaviour
+      description: I think *this* should happen instead.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional Context
+      description: Add any other context about the problem here.
+  - type: textarea
+    attributes:
+      label: Debug Output
+      description: "Run `cargo semver-checks --bugreport`'
```

Comment:
We need to do a release first

Reply:
Looks good to me. I'm in favor of releasing whenever you think is right :+1: